core/src/test/kotlin/io/specmatic/conversions/OpenApiSpecificationTest.kt

```diff
@@ -10975,6 +10975,161 @@ paths:
     }
     
+    @Test
+    fun `HEAD method should be supported in OpenAPI specification parsing`() {
+        val openApiContent = """
+            openapi: 3.0.0
+            info:
+              title: HEAD Method Test API
+              version: 1.0.0
+            paths:
+              /status:
+                head:
+                  summary: Check resource status
+                  responses:
+                    200:
+                      description: Resource exists
+                      headers:
+                        Last-Modified:
+                          schema:
+                            type: string
+                    404:
+                      description: Resource not found
+        """.trimIndent()
+
+        val spec = OpenApiSpecification.fromYAML(openApiContent, "")
+        val feature = spec.toFeature()
+        
+        // Verify that HEAD scenarios are created
+        val headScenarios = feature.scenarios.filter { it.httpRequestPattern.method == "HEAD" }
+        assertThat(headScenarios).isNotEmpty
+        
+        // Verify the HEAD scenario details
+        val headScenario = headScenarios.first()
+        assertThat(headScenario.httpRequestPattern.httpPathPattern?.path).isEqualTo("/status")
+        assertThat(headScenario.httpResponsePattern.status).isIn(200, 404)
+    }
+
+    @Test  
+    fun `HEAD method with examples should be loaded successfully for stub mode`() {
+        val openApiContent = """
+            openapi: 3.0.0
+            info:
+              title: HEAD Method with Examples Test API
+              version: 1.0.0
+            paths:
+              /resource/{id}:
+                head:
+                  summary: Check resource by ID
+                  parameters:
+                    - name: id
+                      in: path
+                      required: true
+                      schema:
+                        type: integer
+                  responses:
+                    200:
+                      description: Resource exists
+                      headers:
+                        Content-Length:
+                          schema:
+                            type: integer
+        """.trimIndent()
+
+        val spec = OpenApiSpecification.fromYAML(openApiContent, "")
+        val (scenarios, stubExamples) = spec.toScenarioInfos()
+        
+        // Verify HEAD scenarios are created
+        val headScenarios = scenarios.filter { it.httpRequestPattern.method == "HEAD" }
+        assertThat(headScenarios).isNotEmpty
+        
+        // Verify scenario has the correct path and method
+        val headScenario = headScenarios.first()
+        assertThat(headScenario.httpRequestPattern.method).isEqualTo("HEAD")
+        assertThat(headScenario.httpRequestPattern.httpPathPattern?.path).isEqualTo("/resource/(id:number)")
+    }
+    
+    @Test
+    fun `HEAD method scenarios should work in test mode`() {
+        val openApiContent = """
```

Comment:
Add test with internal examples (add a path param, then add a named example in request), one where test runs and one where the tests can't run because of an error, and a similar pair for test with an external example.